app/views/nsm/steps/work_item_delete/edit.html.erb

```diff
@@ -1,35 +1,49 @@
 <% title t('.page_title') %>
-<% decision_step_header %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <%= govuk_error_summary(@form_object) %>
     <h1 class="govuk-heading-l"><%= t('.heading') %></h1>
 
     <table class="govuk-table">
-      <caption class="govuk-table__caption govuk-table__caption--m"><%=  @form_object.record.completed_on&.to_fs(:long) || t('.unknown_date') %></caption>
+      <caption class="govuk-table__caption govuk-table__caption--m"></caption>
```

Comment:
If you don't need the `caption` let's remove it entirely - I suspect screen readers etc will get confused by an empty element.